docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -17,12 +13,12 @@ This feature is supported only in 3.0.0-RC1.
      - [Sample directory structure](#sample-directory-structure)
    - [Prerequisites for all domain types](#prerequisites-for-all-domain-types)
    - [Additional prerequisites for JRF domains](#additional-prerequisites-for-jrf-domains)
-   - [Initial use case](#initial-use-case): An initial WebLogic domain
-   - [Update1 use case](#update1-use-case): Dynamically adding a data source using a model ConfigMap
-   - [Update2 use case](#update2-use-case): Deploying an additional domain
-   - [Update3 use case](#update3-use-case): Updating an application in an image
-   - [Cleanup](#cleanup)
-   - [References](#references)
+   - [Initial](#initial-use-case) use case: An initial WebLogic domain
```

Comment:
The 'Inital' and 'Update' use cases are indented differently.
How about doing this instead:
- Use cases:
   - Initial
   - Update1
   - Update2
   - Update3

Reply:
Good idea.